solana/program_v2/packages/client/core/src/AdminService.ts

```diff
@@ -18,23 +19,31 @@ import {
 } from './lib/connection';
 import { NETWORK_SEED, GATEWAY_PROGRAM, SOLANA_MAINNET } from './lib/constants';
 import { GatewayV2 } from '@identity.com/gateway-solana-idl';
-import { AbstractService, ServiceBuilder } from './utils/AbstractService';
+import {
+  AbstractService,
+  NonSigningWallet,
+  ServiceBuilder,
+} from './utils/AbstractService';
 
 export class AdminService extends AbstractService {
   static async build(
     dataAccount: PublicKey,
-    wallet: Wallet,
     cluster: ExtendedCluster = SOLANA_MAINNET,
```

Comment:
You can remove the cluster and customConfig here as well as it is now in the new options parameter. Can check [did-sol](https://github.com/identity-com/sol-did/blob/7dc85f5c41653bfcbe3aafaa6695514329127f55/sol-did/client/packages/core/src/DidSolService.ts#L62) as to how it is handled